modules/swagger-codegen/src/main/resources/go/configuration.mustache

```diff
@@ -27,7 +28,8 @@ func NewConfiguration() *Configuration {
         Debug: false,
         DefaultHeader: make(map[string]string),
         ApiKey: make(map[string]string),
-        ApiKeyPrefix: make(map[string]string) ,
+        ApiKeyPrefix: make(map[string]string),
+        UserAgent: "{{#httpUserAgent}}{{{.}}}{{/httpUserAgent}}{{^httpUserAgent}}Swagger-Codegen/{{{version}}}/go{{/httpUserAgent}}",
```

Comment:
it is inconsistent, in php, it uses `{artifactVersion}}`, anyway, the code has been updated, it worked.